src/shared/components/palette/base-field/field-write.component.ts

```diff
@@ -35,10 +35,11 @@ export class FieldWriteComponent extends AbstractFieldWriteComponent implements
   }
 
   ngOnInit(): void {
-    let componentClass = this.paletteService.getFieldComponentClass(this.caseField, true);
+    const writeMode = !(this.caseField.display_context === 'READONLY')
```

Comment:
This seems a bit odd, why not just 

`const writeMode = this.caseField.display_context !== 'READONLY'`

Also, missing a semicolon

Reply:
@phillip-whitaker-hmcts - yeah... I did raise the same observation earlier, that perhaps `writeMode` is redundant